src/radical/pilot/bootstrapper/default_bootstrapper.sh

```diff
@@ -1210,6 +1229,7 @@ mkdir -p "$VIRTENV"
 echo "VIRTENV : $VIRTENV"
 VIRTENV=`(cd $VIRTENV; pwd -P)`
 echo "VIRTENV : $VIRTENV (normalized)"
+rmdir "$VIRTENV" 2>/dev/null
```

Comment:
It's complicated :P

pip/python do not play together well when one is using a symlinked path, and the other isn't.  So we try to get the normalized path for the virtenv location (`pwd -P`).  If the virtenv exists, that is not a problem - if it does not, it is.  So we create the dir (`mkdir -p` will not complain if it exists), do our check, and remove it again.  The removal will fail on a non-empty dir, ie. on an existing virtenv, so we redirect the error.

An alternative approach would be to do an `if test -d` ...